.github/workflows/TagBot.yml

```diff
@@ -4,6 +4,23 @@ on:
     types:
       - created
   workflow_dispatch:
+    inputs:
+      lookback:
+        default: "3"
+# The following is commented out due to https://github.com/JuliaRegistries/TagBot/issues/388
+# permissions:
+#   actions: read
+#   checks: read
+#   contents: write
+#   deployments: read
+#   issues: read
+#   discussions: read
+#   packages: read
+#   pages: read
+#   pull-requests: read
+#   repository-projects: read
+#   security-events: read
+#   statuses: read
```

Comment:
IMO adding these commented out lines doesn't help much with clarity.
```suggestion
```

Reply:
Perhaps not clarity but the next time someone wants to add these lines because the TagBot documentation says one should, they have a chance of knowing there may be a reason not to.

That's said, we can of course hope that TagBot will be fixed soon (one way or another) then this is moot.

i am fine with either way, choose whichever you prefer